inspirehep/modules/workflows/workflows/edit_article.py

```diff
@@ -43,15 +44,19 @@ def change_status_to_waiting(obj, eng):
 
 def update_record(obj, eng):
     control_number = obj.data['control_number']
-    record = get_db_record('lit', control_number)
-    record.update(obj.data)
-    record.commit()
-
-    user_id = obj.id_user
-    if user_id:
-        _set_transaction_user_id_for_last_record_update(control_number, user_id)
-
-    db.session.commit()
+    if current_app.config.get("ENABLE_INSPIREHEP_REMOTE_RECORD_MANAGEMENT"):
```

Comment:
is this feature flag already in place? all of our feature flags are like ``FEATURE_FLAG_ENABLE_FOO``

Reply:
Yes, this flag is already in place on QA. I know that we are using `FEATURE_FLAG_ENABLE_` prefix but this flag felt too long, Still i can add this prefix to it's name if you wish.